vllm/entrypoints/openai/serving_completion.py

```diff
@@ -428,6 +443,19 @@ async def completion_stream_generator(
                     finish_reason = output.finish_reason
                     stop_reason = output.stop_reason
 
+                    if finish_reason == "error":
+                        logger.error(
+                            "Request-level error for request %s: %s",
+                            request_id,
+                            stop_reason or "unknown",
+                        )
+                        error_data = self.create_streaming_error_response(
+                            "Service temporarily unavailable"
+                        )
```

Comment:
I think we should pass the err_type and status_code here too or else they will default to BadRequest which is incorrect (same for chat completion).